Update dependencies.

davidlehn opened the pull request:
- Update all dependencies.
- Used published packages.
- Update code for newer APIs.
- Switch mock data v1 issuanceDate to v2 validFrom.

The `should fail when "credentialSubject" type is not "BitstringStatusList"` test fails since the change of that type to "ex:InvalidType" causes safe mode violations on terms that were previous type scoped and are now unknown.  How should that test be adjusted?

davidlehn committed:
Update dependencies.

- Update all dependencies.
- Used published packages.
- Update code for newer APIs.
- Switch mock data v1 issuanceDate to v2 validFrom.

dlongley reviewed (approved):
Re: the broken test, add an `@vocab` to the context in the status list VC so those other terms are defined to avoid the safe mode violation. Approving assuming that will cause the test to pass. Thanks!

davidlehn committed:
Fix test.

Use a `@vocab` to work around a test issue where intentionally removing
a type caused type scoped terms to be undefined.

aljones15 commented on lib/index.js:
```diff
@@ -2,16 +2,14 @@
 import {verifyCredential as vcVerifyCredential} from '@digitalbazaar/vc';
 
-const VC_V1_CONTEXT_URL = credentialsCtx.constants.CREDENTIALS_CONTEXT_V1_URL;
+const VC_V1_CONTEXT_URL = vcNamedContexts.get('v1').id;
```
this is a cool feature

aljones15 reviewed (approved):
LGTM

davidlehn committed:
Add missing dev dependency.
